sentry_sdk/integrations/logging.py

```diff
@@ -27,9 +27,10 @@
 
 def ignore_logger(name):
     # type: (str) -> None
-    """This disables the breadcrumb integration for a logger of a specific
-    name.  This primary use is for some integrations to disable breadcrumbs
-    of this integration.
+    """This disables recording (both in breadcrumbs and as events) calls to
+    a logger of a specific name.  Among other uses, many of our integrations
+    use this to prevent their actions being recorded as breadcrumbs. Exposed
```

Comment:
> to prevent their actions being recorded as breadcrumbs

More precisely, the integrations use this to suppress spam breadcrumbs/events from the *framework* that would be sent if the integration was disabled. A classic is sending a stringified traceback as log message, or more generally a log error by the framework for every exception we already recorded via other means

Reply:
Ah, I now understand what *their* means in this context, nvm my comment.